deployment/build.gradle

```diff
@@ -7,6 +7,7 @@
 dependencies {
     bundle project(':business-integrations')
     bundle project(':consumer-solutions')
+    bundle project(':my-bundle')
```

Comment:
Added this so your overriden folder is included in the gw7.

Reply:
got it..